docs/guides/internet-access.md

```diff
@@ -0,0 +1,59 @@
+[](){#ref-guides-internet-access}
+# Internet Access on Alps
+
+The [Alps network][ref-alps-hsn] is mostly configured with private IP addresses (`172.28.0.0/16`).
+Login nodes have public IP addresses which means that they can directly access the internet, while a proxy server provides internet access for compute nodes.
+
+??? info "Compute node proxy configuration"
+
+    Compute nodes are configured with the following environment variables to use the proxy server:
+    
+    ```bash
```

Comment:
To make it easier for users to cut and paste this when they need to, add `export` to the start of each line.

Reply:
On one hand I agree, adding the export makes it easier. On the other, do they need to cut and paste if it's configured correctly? This is really just a "FYI". Which makes me think: do we need to keep the actual values? Or just say that they're set appropriately?

Reply:
I made this comment because I got burnt by exactly this the other day when using a container - the variables don't seem to be set properly inside the container in some cases.
I wanted to cut and paste from the original docs, and was annoyed by having to manually add `export` for each line.

You could create a table with the variables, and an expandable `example` box that has the text ready to "cut and paste"?

Reply:
Fair enough, that's a good reason to leave them here then and add the `export`! I added the `export` in https://github.com/eth-cscs/cscs-docs/pull/39/commits/e5370975c086e777ae159a3b39538ca672d7184a. Is it worth having the extra table as well? I'd prefer not duplicating the information, even if they're close to each other and "easy" to spot if both need to be updated.